bin/pit-init.sh

```diff
@@ -27,6 +27,7 @@ set -e
 ERROR=0
 
 export PITDATA="${PITDATA:-}"
+export SYSTEM_NAME="${SYSTEM_NAME:-0})"
```

Comment:
Don't want this to cause a failure, this var is found-out in the csi function below..

It might be already defined, and I set `0` instead of `''` because I wasn't sure if that would still satisfy  `[ -z "$SYSTEM_NAME" ] && return 1`

Reply:
At what point does this not being set cause an error?  I'm not seeing it on inspection.  `load_csi` is the second thing called... and `init` doesn't reference it.

Reply:
Not sure I follow. If this isn't set in the environment when `pit-init` is called then it fails in the shasta cfg function.

Reply:
Here: https://github.com/Cray-HPE/pit-init/blob/main/bin/pit-init.sh#L145

Reply:
I think the better fix is to not set SYSTEM_NAME in a subshell in load_csi() ... that is why/how it is getting 'lost'

Reply:
I redid this, ready for a re-review. Testing on redbull later/tomorrow.